### PR TITLE
Clarify LLC responsibilities (addresses point 3 from Francesca's discuss)

### PR DESCRIPTION
**Pull request by mirjak**
Based on IESG Review feedback

### CODE REVIEW

File: draft-ietf-shmoo-remote-fee.md [hunk 1]
```diff
@@ -127,15 +127,29 @@ As long as the overall meeting expenses are covered by paid registrations, spons
...
+If the LLC decides to do this, they should make their decision and rationale known to the community
+and consider community consultation as specified in Section 4.4 of RFC8711
+in oder “to obtain consensus-based community input on key issues”. 
+As discussed in the next section, assessment of eligibility is difficult and consequently any limit
+on the number of available free registrations, which likely requires an assessment of eligibility,
+can cause unfairness and
+negatively impact openness which should be considered seriously in any LLC decision.
+This document cannot provide guidance on appropriate measures as any measures need to
+be adapted to the specific problem in a specific situation in order to minimise both the
+financial risk as well as its impact on openness and inclusivity.
```
Comment (larseggert) [inline draft-ietf-shmoo-remote-fee.md:152]: This is just repeating things said elsewhere.

Comment (mirjak) [reply draft-ietf-shmoo-remote-fee.md:152]: I know but I'm trying to address Francesca's and other discuss/abstain points. However, waiting for Francesca to reply.

Comment (mirjak) [reply draft-ietf-shmoo-remote-fee.md:152]: @Francesca can you indicate if the last sentence is needed or not from you point of view?

File: draft-ietf-shmoo-remote-fee.md [hunk 2]
```diff
@@ -127,15 +127,29 @@ As long as the overall meeting expenses are covered by paid registrations, spons
...
+This document does not and cannot restrict the LLC in its financial responsibility
+and therefore does not impose any limitation on the use of appropriate measures.
+If the LLC decides to do this, they should make their decision and rationale known to the community
+and consider community consultation as specified in Section 4.4 of RFC8711
+in oder “to obtain consensus-based community input on key issues”. 
+As discussed in the next section, assessment of eligibility is difficult and consequently any limit
+on the number of available free registrations, which likely requires an assessment of eligibility,
+can cause unfairness and
+negatively impact openness which should be considered seriously in any LLC decision.
+This document cannot provide guidance on appropriate measures as any measures need to
```
Comment (mirjak) [inline draft-ietf-shmoo-remote-fee.md:150]:
```suggestion
This document defines the principle of free participation but leaves room for choices in the implementation. Specifically, it cannot provide guidance on appropriate measures against misuse as any measures need to
```
Adding even more words to make think even more clear but not sure if that helps...

File: draft-ietf-shmoo-remote-fee.md [hunk 3]
```diff
@@ -139,16 +139,34 @@ As long as the overall meeting expenses are covered by paid registrations, spons
...
+participants to make an informed decsion about the use of the free option.
+As discussed in the next section, assessment of eligibility is difficult and consequently any limit
+on the number of available free registrations, which likely requires an assessment of eligibility,
+can cause unfairness and
+negatively impact openness which should be considered seriously in any LLC decision.
+This document defines the principle of free participation but leaves
+room for choices in the implementation. Specifically, it cannot provide
+guidance on appropriate measures against misuse as any measures need to
+be adapted to the specific problem in a specific situation in order to minimise both the
+financial risk as well as its impact on openness and inclusivity.
```
Comment (mirjak) [inline draft-ietf-shmoo-remote-fee.md:169]:
```suggestion

As discussed in the next section, assessment of eligibility is difficult and consequently any limit
on the number of available free registrations, which likely requires an assessment of eligibility,
can cause unfairness and negatively impact openness which should be considered seriously in any LLC decision.
As such, this document defines the principle of free participation but leaves
room for choices in the implementation. Specifically, it cannot provide
guidance on appropriate measures against misuse as any measures need to
be adapted to the specific problem in a specific situation in order to minimise both the
financial risk as well as its impact on openness and inclusivity.
```

**Pull request merged**